keybase/report.go

```diff
@@ -52,6 +52,8 @@ func (c context) reportAction(action updater.UpdateAction, options updater.Updat
 	data.Add("version", options.Version)
 	data.Add("upd_version", options.UpdaterVersion)
 	data.Add("action", action.String())
+	autoUpdate, _ := c.config.GetUpdateAuto()
```

Comment:
yeah, the autoUpdate config is set/saved immediately after the prompt returns... and the action is reported after this... so we should be all good... I need to add more tests for this though